types/priv_validator.go

```diff
@@ -17,6 +18,40 @@ type PrivValidator interface {
 
 	SignVote(chainID string, vote *cmtproto.Vote) error
 	SignProposal(chainID string, proposal *cmtproto.Proposal) error
+	SignRawBytes(chainID, uniqueID string, rawBytes []byte) ([]byte, error)
+}
+
+// RawBytesSignBytesPrefix defines a unique prefix added to raw bytes for signing purposes.
+const RawBytesSignBytesPrefix = "COMET::RAW_BYTES::SIGN"
```

Comment:
It might be good to clarify the actual purpose of this prefix:

```suggestion
// RawBytesSignBytesPrefix defines a domain separator prefix added to raw bytes to ensure the resulting
// signed message can't be confused with a consensus message, which could lead to double signing
const RawBytesSignBytesPrefix = "COMET::RAW_BYTES::SIGN"
```